doc/2/guides/advanced/configuration/index.md

```diff
@@ -44,6 +44,16 @@ For example, the `.kuzzlerc` parameter `services.storageEngine.host` in example
 export kuzzle_services__storageEngine__host="http://localhost:9200"
 ```
 
+You can also pass stringified JSON values this way to override non-scalar values such as objects or arrays.  
+To do so, prefix a valid stringified JSON with `*json:` to instruct Kuzzle to parse the content of the value as JSON.
```

Comment:
Why `*json` as placeholder ? (the `*`)

Reply:
I think the is to avoid conflict with parameters name `jsonSomething`